src/core/Header/HeaderButton.test.tsx

```diff
@@ -149,7 +149,7 @@ it('should render menuItems correctly', () => {
   );
 
   const button = container.querySelector(
-    '.iui-header-button.iui-dropdown.iui-borderless',
+    '.iui-header-button.iui-dropdown.iui-borderless.iui-header-dropdown-button',
```

Comment:
Can we just grab `iui-header-dropdown-button` here instead of all of these?

Reply:
done